### DISCUSSION TITLE
Surface Icon

### DISCUSSION THREAD

**Discussion by nebsar** [pelicanmapping/osgearth#1949] [Support Q&A]
How can I draw a 2D surface icon such as an aircraft icon, on the ground with heading and lat lon location can be set?

**Comment by gwaldron** [comment]
What do you mean? Can you provide more details or a picture?

**Reply by nebsar** [reply]
I mean something like in the image below. For a moving map.

![image](https://user-images.githubusercontent.com/5169314/151820917-6470d5c6-119c-453f-aab7-3bc579ca9ff2.png)

Thanks Glenn

**Comment by nebsar** [comment]
I think icons can be drawn as track node or place node. But I need to change heading of the icon on the fly. How can I do it?
